patches/series

```diff
@@ -1,5 +1,4 @@
-# This series applies on GIT commit v2.9.0
+# This series applies on GIT commit 8cadc751ca6cccbf2b8edbab6b11c01b347418eb
 unix-socket.patch
 set-config-api.patch
 monitor-table-multi.patch
-generated-grpc.patch
```

Comment:
Do we no longer want generated-grpc.patch?  How do we know that?  If we don't, is there a reason to keep the .patch file around?

Reply:
Apologies for not explaining this earlier: This patch merely updates the "generated" proto.go file from a previous proto patch. But you should never be modifying proto.go files directly. Instead, I undo the patch and add to the build script a step to "build" proto.go out of the proto changes, and do this BEFORE building all the go code. This is arguably correct protocol buffer behavior.